ts/parts/Tooltip.ts

```diff
@@ -247,6 +247,8 @@ declare global {
  * @typedef {"callout"|"circle"|"square"} Highcharts.TooltipShapeValue
  */
 
+''; // separates doclets above from variables below
```

Comment:
?

Reply:
Yeah. That was interesting. All the doclets above where attached to the variable below, which was internal and filtered out. Therefor no interface was going into the class reference. :)

Reply:
As I understand it is a hack to make sure that the doclets is picked up by JSDoc without the variables below interfering with the result.

Maybe you can confirm or correct me on this @bre1470?

Reply:
I guess simple:

// ---- doclets separator ----

Won't work? :) Maybe it's better to place doclets at the end of the file? And make this a general rule? I'm worried that in future someone else will spot the same issue, wasting time to find answer "whyyyyy?"